bayesian/default_config.py

```diff
@@ -58,3 +58,8 @@
 COMPONENT_ANALYSES_LIMIT = int(os.getenv('COMPONENT_ANALYSES_LIMIT', 10))
 
 STACK_ANALYSIS_REQUEST_TIMEOUT = int(os.getenv('STACK_ANALYSIS_REQUEST_TIMEOUT', 120))
+
+THREESCALE_ACCOUNT_SECRET = os.getenv('THREESCALE_ACCOUNT_SECRET', "")
+BAYESIAN_API_HOSTNAME = os.getenv('BAYESIAN_API_HOSTNAME', 'http://recommendationapi-server:5000')
+STACK_REPORT_UI_HOSTNAME = os.getenv('STACK_REPORT_UI_HOSTNAME',
+                                     'https://stack-analytics-report.openshift.io')
```

Comment:
Please add corresponding entries in template.yaml

Reply:
Ack